packages/twenty-docs/docs/user-guide/user-guide.mdx

```diff
@@ -12,7 +12,7 @@ sidebar_custom_props:
 
 The purpose of this user guide is to help you learn how you can use Twenty to build the CRM you want. 
 
-This quick-start guide walks you through the basics.
+This quick-start guide walks you through the basics you need to know to start using Twenty to its full potential in no time.
```

Comment:
Feels a bit duplicate with sentence above, I don't feel like the longer version is better. Straight to the point = better (imo).